packages/manager/modules/log-to-customer/src/live-tail/controller.js

```diff
@@ -25,20 +31,100 @@ export default class LogToCustomerCtrl {
       createAccount: `${this.dbaasLogsURL}/order`,
     };
 
+    this.kind = this.kindInitValue;
     this.hasAtLeastOneDataStream = false;
+    this.errorMessage = null;
 
-    this.kind = this.kindInitValue;
+    this.init();
+  }
 
-    this.LogToCustomer.getDataStreams()
-      .then(({ streams, streamCount }) => {
-        this.logAccountStreams = streams;
-        this.hasAtLeastOneDataStream = streamCount > 0;
+  init() {
```

Comment:
`init` name doesn't make sense as you already have a `$onInit` function and values initialization are done in both functions. You should define a more accurate name to explain the purpose
```suggestion
  loadStreams() {
```